plugins/Bitcrush/Bitcrush.cpp

```diff
@@ -35,8 +35,8 @@ const int OS_RATE = 5;
 const float OS_RATIO = 1.0f / OS_RATE;
 const float CUTOFF_RATIO = 0.353553391f;
 const int SILENCEFRAMES = 10;
-const float OS_RESAMPLE [5] = { 0.0001490062883964112, 0.1645978376763992, 0.6705063120704088,
-		0.1645978376763992, 0.0001490062883964112 };
+const auto OS_RESAMPLE = std::array{0.0001490062883964112f, 0.1645978376763992f, 0.6705063120704088f,
```

Comment:
Explicitly specifying this as a std::array of floats would probably be better than inferring that from integer suffixes.